awscli/compat.py

```diff
@@ -37,6 +37,12 @@
     ZIP_COMPRESSION_MODE = zipfile.ZIP_STORED
 
 
+try:
+    import sqlite3
+except ImportError:
+    sqlite3 = None
```

Comment:
This will happen before debug logs are configured though so I don't think it'd ever show up anywhere.  Plus you log if sqlite3 isn't available when you try to enable the customization.